cli/azd/pkg/exec/runargs.go

```diff
@@ -29,6 +31,14 @@ type RunArgs struct {
 
 	// When set will call the command with the specified StdIn
 	StdIn io.Reader
+
+	// When set will call the command with the specified StdOut
+	StdOut io.Writer
+}
+
+type WithPreviewOptions struct {
+	Console        input.Console
+	PreviewOptions *input.ShowPreviewerOptions
 }
```

Comment:
Lets just change this to `WithStdOut` like we use for other builder options on the `RunArgs`.

Reply:
this `WithPreviewOptions` is not used any more.. removing..